beyond-essence/inventory/links.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Links
+background_image: index.jpg
+---
+
+Links are navigation elements.
+They are interactive and give users access to further content that is not directly accessible via the primary navigation.
+
+## Use cases
+
+In the Beyond cockpit, we provide various types of links:
+
+* Links in tables and overviews
+
+{% image_custom image="/assets/img/pages/essence/links-in-tables.png" width="50" lightbox %}
```

Comment:
Proposal: We could distinguish a little bit more. Described example above is an example for `Links in tables` and we also could have a section `Links in overviews`:
<img width="994" alt="Bildschirmfoto 2019-08-09 um 14 36 26" src="https://user-images.githubusercontent.com/6104802/62779297-183bdf00-bab3-11e9-82f0-20a6fbd11efa.png">
I know from a technical point of view both examples are tables so it is more a visual perception.